docs/markdown/recursor/settings.md

```diff
@@ -23,6 +23,7 @@ queries needed.
 ## `allow-from`
 * IP ranges, separated by commas
 * Default: 10.0.0.0/8, 172.16.0.0/12, 192.168.0.0/16
+* Available since: 2.9.21
```

Comment:
True, dunno what happened, though. I just double-checked my output and seemingly (for whatever reason) that one must've slipped through. I am gonna correct my PR.